docs/contents.md

```diff
@@ -0,0 +1,208 @@
+
+The general gist is that, as your package travels from your repository to end
+users machines, various files probably can (and should) be removed. This is
+done to:
+
+1. Keep [this repository][this_repo] small and
+2. Reduce downloads (and disk usage) for end users
+
+
+## Table of contents
+
+1. [Categories of files][a]
+    1. [Required files][files_required]
+    1. [Documentation files][files_docs]
+    1. [Other files][files_other]
+1. [Package development version][b]
+1. [Published package version][c]
+1. [Imported package version][d]
+
+[a]: #categories-of-files
+[files_required]: #required-files
+[files_docs]: #documentation-files
+[files_other]: #other-files
+[b]: #package-development-version-stage-1
+[c]: #published-package-version-stage-2
+[d]: #imported-package-version-stage-3
+
+
+## Categories of files
+
+The files possibly contained in your package are broadly grouped into the
+following three categories.
+
+
+### Required files
+
+Files that are necessary for the package to work. If any of these files are
+removed, the package would break for the end user.
+
+This includes the manifest file (`typst.toml`), main Typst file (mentioned
+under `package.entrypoint` in `typst.toml`) and its dependencies, and in case
+of a template package, any file in the template directory. On top of this, the
+following files are also included:
+
+- The LICENSE file (e.g. `LICENSE` or `COPYING`): This should always be
+  available along with the source code
+- The README file (e.g. `README.md`): This is generally a lightweight file, and
+  can provide minimal documentation in case the user is offline or can't access
+  anything else than their local package cache for some other reason
+
+*Examples*:
+
+- `README.md`
+- `LICENSE`
+- `typst.toml`
+- `src/lib.typ` (see `package.entrypoint` in `typst.toml`)
+- `my-plugin.wasm` (for WASM plugins)
+- Images/icons/other assets if your package reproduces these in documents
+
+
+### Documentation files
+
+Files that are necessary for the package to be displayed correctly on [Typst
+Universe][typst_universe].
+
+This includes any files that are linked from the README (manuals, examples,
+illustrations, etc.). These files can easily be accessed by opening the package
+README. If your README links to directories (such as `examples/`), include
+these directories as well. In the case of PDF manuals, for example, it suffices
+to include the built PDF without the source code.
+
+*Examples*:
+
+- `docs/user-manual.pdf` (but *not* the source code, if any)
+- `assets/screenshot.png` (if your README links to that)
+- `examples/try-this.typ` (if your README links to that)
+
+
+### Other files
+
+This generally includes test files, build scripts, but also examples or manuals
+that are not linked in the README.
+
+These files would be almost impossible to access for the final user, unless
+they browse [this GitHub repository][this_repo] or their local package cache.
```

Comment:
This is not correct, since these files should not be included in the final archive and thus should never be stored in the cache. Can you remove the end of the sentence, starting at "or"?

Reply:
Well, in a way I think it is, basically, correct. It's just that we don't want this to be the case, right?

If I were to copy my bare package contents (as taken from my Git forge of choice) into this repository and somehow it passed review, the files would in fact be here. And they would show up in the archive that users download (assuming i haven't configured `package.exclude`). It's just that due to how Typst Universe and the actual package ecosystem (using `#import`) interact with the code I upload here, those files will be invisible to most users. If you insist I can still remove it, but I think as it is (and I took that verbatim from the previous docs in `tips.md`) this helps users understand *why* we don't want those files to be here.